src/main/java/taxi/controller/user/LogoutController.java

```diff
@@ -0,0 +1,18 @@
+package taxi.controller.user;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = "/logout")
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getSession().invalidate();
+        resp.sendRedirect("/index");
```

Comment:
```suggestion
        resp.sendRedirect("/login");
```
how can user access /index after logout?

Reply:
anyway it will go through filter and will be redirected to login